### DISCUSSION TITLE
Aktaramadığım puanlar var

### DISCUSSION THREAD

**Discussion by Engintunc1907** [IBAX-io/go-ibax#789] [Soru Cevap - Türkçe]
Merhaba, 10000 puanı içe aktardım ama 8500 civarında puanım var aktaramıyorum bir türlü :(

**Comment by Engintunc1907** [comment]
![Uploading Screenshot_20220125-225431_Telegram.jpg…]()

**Comment by Yusufknk** [comment]
Puanların karşılığı 1 Şubat'tan sonra cüzdanlara dağıtım için aktarılacaktır.

**Reply by Engintunc1907** [reply]
Dostum teşekkür ederim ama şu görsellerle de anlatmak sormak öğrenmek istiyorum müsadenle. Doprumanlatanilmişmiyim bir bakarmısın? 

![Screenshot_20220125-230515_Gallery](https://user-images.githubusercontent.com/98422964/151051237-2c2de79b-8fa1-4046-bf9e-3aef953d4f34.jpg)
Z
![Screenshot_20220125-230436_Chrome](https://user-images.githubusercontent.com/98422964/151051291-a61c8f34-21a4-48de-81d5-7ba9a1921f5c.jpg)

**Reply by Yusufknk** [reply]
Yapılmayan retweet görevleriniz mevcut. Birde cüzdanın bağlandığından emin olun (My dashboard-retwet)

**Reply by Engintunc1907** [reply]
Teşekkür ederim kontrol edeceğim.
